src/arrays/ArrayUtil.java

```diff
@@ -36,59 +49,101 @@ public static void create2dArray(int x, int y) {
      * @return true if the array contains the given value, otherwise return false.
      */
     public static boolean contains(int[] array, int value) {
-
-        return false;
+        boolean bool = true;
+        for(int val: array){
+            if (val == value){
+                bool = true;
+                break;
+            }else {
+                bool = false;
+            }
+        }
+        return bool;
     }
 
+
     /**
      * @param array An array to search in.
      * @param value The value to search for.
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+        int bool = array[0];
```

Comment:
this will not work properly.

Reply:
you just returning first element of the array, you need to return the index of given element